src/misclibs.F

```diff
@@ -4093,46 +3957,24 @@ subroutine totmois(nstat,rstat,train,ruh,rvh,rmh,qv,ql,qi,rho)
  
       integer i,j,k
       double precision :: tmass,var
-      double precision, dimension(nk) :: foo
       double precision :: foot
 
-#ifdef _B4B07R
-      !$acc update host(rho,qv,ql,qi,ruh,rvh,rmh)
-#else
-      !$acc data create(foo)
-
+      tmass=0.0d0
       !$omp parallel do default(shared) private(i,j,k)
-      !$acc parallel default(present) private(i,j,k) reduction(+:foot)
+      !$acc parallel default(present) private(i,j,k) reduction(+:foot,tmass)
       !$acc loop gang
```

Comment:
Do we need to add `reduction(+:tmass)` here?

Reply:
Good catch, yes we do.